client/ayon_core/hosts/houdini/api/pipeline.py

```diff
@@ -262,7 +262,29 @@ def parse_container(container):
         dict: The container schema data for this container node.
 
     """
-    data = lib.read(container)
+    # Read only relevant parms
+    # TODO: Clean up this hack replacing `lib.read(container)`
```

Comment:
@antirotor This might be up for discussion - but data used from a container usually is only these required fields. By parsing only these instead of **all** spare parameters of a node it may greatly reduce the amount of values returned and thus the speed and memory usage of this call. E.g. on nodes with Redshift parameters all the redshift related attributes are "spare parameters" and thus 100s of parms will be read otherwise.

Let me know what you think - of course we'll need to clean up the todo. ;)

Reply:
uh, I've missed that one, but sure - it make sense to read just what we need

Reply:
This has worked out fine in our pipeline as we've been using this in production over the past months. I'll resolve this conversation; the TODO is there to see if we should clean up elsewhere over time.